CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ When you're ready to submit your contribution, you're going to create a [pull re
 
 If this is your first time, or you need a refresher on how to create a PR, you can check out this video:
 
-[How to Contribute to an Open Source Project on GitHub}](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
```

Comment:
This change is unrelated, I just noticed a typo and bundled the fix in.

Please note that there are other changes in this file, all about trailing whitespace. 

I have my editor configured to trim trailing whitespace on save. I would suggest we all do the same to avoid these messy diffs. :wink:

Reply:
ack! just turned it on :)